### PR TITLE
Add note about preloading data

### PR DESCRIPTION
**Pull request by RichardLitt**
See https://github.com/orbitdb/orbit-db/issues/560.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -49,6 +49,19 @@ However, this will change in the future as js-ipfs gets GC and we want to make s
...
+This happens when there's only one node with data available, and the system isn't able to effectively get all of the data it needs from it. in order to get around this, instantiate IPFS with preload enabled:
+
+```js
+this.ipfs = new Ipfs({
+  preload: { enabled: false },
+  // ...
+}
+```
+
+This will preload data on the remote nodes, spreading out the demand across multiple nodes to avoid hitting one node too much.
```
Comment (aphelionz) [inline FAQ.md]: This is correct but we're actual _disabling_ preload in this instance because the preload nodes are sometimes down.

Comment (RichardLitt) [reply FAQ.md]: I should have actually read that. Edited and pushed change.

**Comment by aphelionz** [comment]
Much better, thank you!

**Comment by RichardLitt** [comment]
@haadcode What do you suggest as a way to merge FAQ questions faster? How about one accept is enough?

**Pull request merged**